servers/dsub/Dockerfile

```diff
@@ -16,4 +16,4 @@ RUN rm -rf jm_utils
 
 # Missing required arguments -b PORT, -e ... which must be provided by the
 # docker image user.
-ENTRYPOINT ["/env/bin/gunicorn", "jobs.__main__:app"]
+ENTRYPOINT ["/bin/bash", "/scripts/await_md5_match.sh", "/app/jobs/models/.jobs.yaml.md5", "--", "/env/bin/gunicorn", "jobs.__main__:app"]
```

Comment:
This is awkward in the `Dockerfile`. I think I ought to remove `ENTRYPOINT` altogether and set the entire command in the `docker-compose.yml`

Reply:
Agreed - for dsub we do currently use this Dockerfile for deployment, which sort of depends on the entrypoint being setup like this (not sure about the Cromwell shim). I think for this specific Dockerfile, you can just leave it as is (perhaps adding a call to run the swagger generation a few lines up).

For the other local dev Dockerfiles, perhaps just remove the entrypoints and move the logic into docker-compose entrypoint or command?

Reply:
@rexwangcc should probably verify the changes to the Cromwell shim Dockerfile; I'm not sure what image k8s is actually using.

Reply:
Mint's k8s cluster is using the dockerfile `servers/cromwell/Dockerfile`. When removing the `ENTRYPOINT`, please make sure add docs about it, otherwise the person who deploys the docker image has no idea what command to add to either docker-compose file or kubernetes yaml file.